### PR TITLE
DOC: Fix Jarque-Bera documentation

### LINKED ISSUE #1579: Jarque-Bera does not include null in the docstring
Consider closing as part of #1182.

### PR DESCRIPTION
**Pull request by bashtage**
Improve Jarque-Bera documentation

closes #1579

[skip ci]

### CODE REVIEW

File: statsmodels/stats/stattools.py [hunk 1]
```diff
@@ -76,27 +76,41 @@ def omni_normtest(resids, axis=0):
...
+    data : array-like
+        Data to test for normality
     axis : int, optional
-        Default is 0
+        Axis to use if data has more than 1 dimension. Default is 0
 
     Returns
     -------
-    JB, JBpv, skew, kurtosis
+    out : tuple (JB, JBpv, skew, kurtosis)
```
Comment (josef-pkt) [inline statsmodels/stats/stattools.py]: python always returns a tuple when there are several return values. So this should be standard format with `JB, JBpv, skew, kurtosis` separately documented as returns

File: statsmodels/stats/stattools.py [hunk 2]
```diff
@@ -105,7 +119,7 @@ def jarque_bera(resids, axis=0):
 
     # Calculate the Jarque-Bera test for normality
     n = resids.shape[axis]
-    jb = (n / 6.) * (skew**2 + (1 / 4.) * (kurtosis - 3)**2)
+    jb = (n / 6.) * (skew ** 2 + (1 / 4.) * (kurtosis - 3) ** 2)
```
Comment (josef-pkt) [inline statsmodels/stats/stattools.py]: (I'm giving up on "no spaces around `**`", given that pep-8 enforcers seems to become stronger )

File: statsmodels/stats/stattools.py [hunk 3]
```diff
@@ -76,27 +76,42 @@ def omni_normtest(resids, axis=0):
...
 
-    JB = n/6*(S^2 + (K-3)^2/4)
+    Notes
+    -----
+    Each output returned has 1 dimension fewer than data
 
-    JBpv is the Chi^2 two-tail probability value
+    The Jarque-Bera test statistic tests the null that the data is normal
+    against an alternative that it it is not. It is based on two moments of
+    the data, the skewness and the kurtotis, and has an asymptotic
```
Comment (josef-pkt) [inline statsmodels/stats/stattools.py]: typo kurtosis   (s instead of second t)

File: statsmodels/stats/stattools.py [hunk 4]
```diff
@@ -76,27 +76,42 @@ def omni_normtest(resids, axis=0):
...
+        Estimated kurtosis of the data
 
-    JB = n/6*(S^2 + (K-3)^2/4)
+    Notes
+    -----
+    Each output returned has 1 dimension fewer than data
 
-    JBpv is the Chi^2 two-tail probability value
+    The Jarque-Bera test statistic tests the null that the data is normal
+    against an alternative that it it is not. It is based on two moments of
```
Comment (josef-pkt) [inline statsmodels/stats/stattools.py]: I would add "distributed" after normal. AFAICS the entire docstring doesn't mention "distribution"

**Comment by josef-pkt** [comment]
two minor comments, otherwise looks good to merge.

**Comment by bashtage** [comment]
Amended commit to keep log small.

**Comment by josef-pkt** [comment]
Thanks, and thanks for the comment which triggered a github notification.

merging

**Pull request merged**